packages/serializer/addon/-private/embedded-records-mixin.js

```diff
@@ -19,6 +19,8 @@ import { warn } from '@ember/debug';
 
   Note that embedded records will serialize with the serializer for their model instead of the serializer in which they are defined.
 
+  Note also that this mixin does not work with JSONAPISerializer because the JSON API spec does not describe how to format embedded resources.
```

Comment:
General question I do not know the answer to: what versions of the JSON:API spec does ember-data adhere to? I would like to qualify this caveat with that answer.

Reply:
![image](https://user-images.githubusercontent.com/5004319/72270719-9ae14100-35f3-11ea-891b-b2629717744e.png)

Reply:
Thanks! Upon further reflection I think ember-data should describe its compatibility with JSON:API somewhere for transparency. It seems better suited to top-level documentation or some kind of introduction. So we can leave it out of this PR.

Reply:
> Upon further reflection I think ember-data should describe its compatibility with JSON:API somewhere for transparency. It seems better suited to top-level documentation or some kind of introduction. 

We do in the serializer documentation overview